docs/agent-reference/how-to-run-agent.md

```diff
@@ -1,5 +1,16 @@
 # Running the Device Update for IoT Hub Reference Agent
 
+## Required Dependencies
+
+* Install the [Delivery Optimization SDK](https://github.com/microsoft/do-client)
+* Install the [Delivery Optimization components](https://github.com/microsoft/do-client#installing-do-client-components) i.e., the Delivery Optimization agent & Delivery Optimization APT plugin 
+
+Alternatively you can add the Microsoft Repository to your local APT repositories using these (instructions)[https://docs.microsoft.com/windows-server/administration/linux-package-repository-for-microsoft-software]. Then you will be able to install the necessary DO dependencies using
+
+```bash
+sudo apt-get install deliveryoptimization-plugin-apt
+```
+
```

Comment:
This is only true for Ubuntu 18.04 amd64.

For these set of docs, which are instructing users how to build and install DU + DO from source. I would remove any mention of packages.

Reply:
Updated to:
Follow the instructions from this page to build and install the Delivery Optimization agent and plug-in. Note: Delivery Optimization SDK is already installed as part of step 1 above.